blocks/components/editable/index.js

```diff
@@ -319,6 +388,27 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	isEmpty( value ) {
+		// Empty array.
```

Comment:
Can we assume that `value` is always an array? I think based on our usage of `wp.blocks.query.children`, this would always be the case, but are we sure values are only ever passed from the result of `children`, not just arbitrary React elements?

I think I'd actually be open to standardizing though, since we've been encountering more and more issues where value shapes vary depending on if there's on child, many children, etc. So I could be favor of this assumption, as long as it's well enforced.